tests/attacks/test_carlini.py

```diff
@@ -42,7 +43,6 @@ class TestCarlini(TestBase):
 
     @classmethod
     def setUpClass(cls):
-        master_seed(seed=1234)
```

Comment:
Why do you remove this line?

Reply:
because that is now located in only one location in the entire test framework for ART instead of each tests. You can find it in the main conftest.py file on line 17